### PR TITLE
feat[added Navbar]

### PR DESCRIPTION
**Pull request by Radheshyam1234**
front-kart-navbar

### CODE REVIEW

File: src/Components/Header/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  cursor: pointer;
+}
+
+.suggestion-item {
+  padding: 8px 1.5rem;
+  cursor: pointer;
+}
+.suggestion-item:hover {
+  color: white;
+  background-color: #c01322;
```
Comment (pratik-kumar-sahu) [inline src/Components/Header/Navbar.css:20]: - You can use variables for colors

**Review by pratik-kumar-sahu — APPROVED** [review]
- Its nice that you are using debouncing for search. 
- Looks good to merge. Go ahead.

File: src/Components/Header/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.search-data-results {
+  position: absolute;
+  /* top: calc(2.4rem); */
```
Comment (junaidshaikh-js) [inline src/Components/Header/Navbar.css:3]: code comments should be removed

File: src/Components/Header/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import React from "react";
```
Comment (junaidshaikh-js) [inline src/Components/Header/Navbar.js:1]: From React 17 you don't need to import React anymore

File: src/Components/Header/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+import { useAuthProvider } from "../../Context/AuthContext/AuthProvider";
+import { truncateString } from "../../utilities/turncateString";
+import "./Navbar.css";
+
+export const Navbar = () => {
+  const { userState } = useStateProvider();
+  const { isUserLoggedIn, userProfile } = useAuthProvider();
+  return (
+    <header className="header">
+      <i className="fas fa-bars hamburger-menu"></i>
```
Comment (junaidshaikh-js) [inline src/Components/Header/Navbar.js:14]: Add titles to you icons for better accessibility

File: src/Components/Header/SearchBox.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    let timerId;
+    return function () {
+      let self = this;
+      let args = arguments;
+      clearTimeout(timerId);
+      timerId = setTimeout(() => {
+        searchHandler.apply(self, args);
+      }, delay);
+    };
+  }
```
Comment (junaidshaikh-js) [inline src/Components/Header/SearchBox.js:62]: Good use of debounce for search implementation

**Review by junaidshaikh-js — APPROVED** [review]
Great work! 

Suggested some minor changes. 

Good use of debounce and utility functions. Can be merged.

**Pull request merged**